homeassistant/components/light/zwave.py

```diff
@@ -244,9 +235,9 @@ def _value_added(self, value):
         # Check for the missing color values
         self._get_color_values()
 
-    def update_properties(self):
+    def update(self):
```

Comment:
Beware that because we are doing multiple inheritance, that `update` might be overwritten from the Component ABC instead of the ZWaveEntityDevice